src/styles/main.css

```diff
@@ -69,7 +69,7 @@
 /* Button */
 
 .button {
-    @apply uppercase font-sans text-sm leading-none py-2 px-6 w-40 h-10;
+    @apply uppercase font-sans text-sm text-center leading-none py-2 px-6 w-40 h-10;
```

Comment:
@alissonfpmorais  awesome!!! @lucca65  this fixes the explore button text in English 
![Screen Shot 2019-09-23 at 08 06 31](https://user-images.githubusercontent.com/897731/65404509-13b85500-ddd9-11e9-9c51-f8ad26b9113e.png)